po/de.po

```diff
@@ -4627,9 +4628,9 @@ msgid "sparse:path filters support has been dropped"
 msgstr "Keine Unterstützung für sparse:path Filter mehr"
 
 #: list-objects-filter-options.c:105
-#, fuzzy, c-format
+#, c-format
 msgid "'%s' for 'object:type=<type>' isnot a valid object type"
```

Comment:
Hier fehlt ein Leerzeichen. Das wurde aber schon behoben.

Reply:
dann wird es sicherlich in einer "round 2" noch ein update dazu geben... noch gibt es im upstream master nichts neues..